apps/api/src/app/events/services/workflow.queue.service.ts

```diff
@@ -143,6 +147,19 @@ export class WorkflowQueueService {
       return false;
     }
 
+    const where: any = {
```

Comment:
I think there is a type here you could import, otherwise you can add `Partial<JobEntity>` or something similar :)